### ISSUE TITLE
Failed to activate the language-documentation package

### ISSUE THREAD

**Issue by robksawyer** [DHedgecock/language-documentation#8]
This error occurred after simply installing the package.

1. Search for package
2. Install

**Atom**: 1.16.0 x64
**Electron**: 1.3.13
**OS**: Mac OS X 10.12.4
**Thrown From**: [language-documentation](https://github.com/DHedgecock/language-documentation) package 2.0.0

### Stack Trace

Failed to activate the language-documentation package

```
At Cannot read property 'unshift' of undefined

TypeError: Cannot read property 'unshift' of undefined
    at /packages/language-documentation/lib/language-documentation.js:33:25
    at Array.forEach (native)
    at injectPatterns (/packages/language-documentation/lib/language-documentation.js:28:43)
    at /packages/language-documentation/lib/language-documentation.js:101:7
    at Array.forEach (native)
    at Object.activate (/packages/language-documentation/lib/language-documentation.js:99:33)
    at Package.module.exports.Package.activateNow (/app.asar/src/package.js:218:19)
    at /app.asar/src/package.js:190:32
    at Package.module.exports.Package.measure (/app.asar/src/package.js:96:15)
    at /app.asar/src/package.js:183:26
    at Package.module.exports.Package.activate (/app.asar/src/package.js:180:34)
    at PackageManager.module.exports.PackageManager.activatePackage (/app.asar/src/package-manager.js:550:34)
    at /app.asar/node_modules/settings-view/lib/package-manager.js:484:29
    at exit (/app.asar/node_modules/settings-view/lib/package-manager.js:99:16)
    at triggerExitCallback (/app.asar/src/buffered-process.js:322:11)
    at /app.asar/src/buffered-process.js:343:11
    at /app.asar/src/buffered-process.js:204:9)
    at emitOne (events.js:101:20)
    at Socket.emit (events.js:188:7)
    at Pipe._handle.close [as _onclose] (net.js:493:12)
```

### Commands

```
     -3:52.8.0 editor:duplicate-lines (input.hidden-input)
     -3:52 core:undo (input.hidden-input)
     -3:51.5.0 core:save (input.hidden-input)
     -3:49.5.0 jsdoc-preview:toggle (input.hidden-input)
     -3:39.8.0 command-palette:toggle (input.hidden-input)
  2x -3:37.4.0 core:move-down (input.hidden-input)
     -3:36.8.0 core:confirm (input.hidden-input)
     -3:36.8.0 settings-view:install-packages-and-themes (input.hidden-input)
     -3:32.3.0 core:confirm (input.hidden-input)
     -3:18.2.0 core:backspace (input.hidden-input)
     -3:17.6.0 core:confirm (input.hidden-input)
  2x -3:13.2.0 core:backspace (input.hidden-input)
  2x -3:12.5.0 core:confirm (input.hidden-input)
     -1:31.8.0 core:move-right (input.hidden-input)
     -1:31 editor:delete-to-beginning-of-word (input.hidden-input)
     -1:26.8.0 core:confirm (input.hidden-input)
```

### Non-Core Packages

```
angular-2 0.2.6 
angularjs 0.4.0 
arduino-upload 1.0.3 
atom-autocomplete-php 0.22.2 
atom-beautify 0.29.23 
atom-jade 0.3.0 
atom-typescript 11.0.3 
atomatigit 1.5.5 
autocomplete-clang 0.11.3 
autocomplete-ionic-framework 0.2.2 
autocomplete-ionic2-framework 0.3.2 
autocomplete-python 1.8.63 
browser-plus 0.0.87 
build 0.67.0 
build-xcodebuild 0.2.0 
busy 0.7.0 
busy-signal 1.4.1 
change-case 0.6.5 
coffeedocs 1.0.0 
color-picker 2.2.5 
colorful-json 1.1.0 
docblockr 0.9.4 
docker 0.7.0 
Extempore-Atom 0.3.2 
file-icons 2.1.4 
formatter-coffeescript 0.1.1 
g-code 1.1.0 
git-plus 7.7.0 
highlight-bad-chars 1.0.5 
html-entities 0.4.0 
intentions 1.1.2 
ionic-framework-snippets 1.2.1 
ionic-preview 0.7.0 
jade 1.1.0 
jscs-fixer 1.3.0 
language-arduino 0.4.3 
language-babel 2.58.0 
language-diff 0.7.0 
language-documentation 2.0.0 
language-ejs 0.4.0 
language-hcl 0.4.0 
language-ini 1.19.0 
language-javascript-jsx 0.3.7 
language-linuxcnc-gcode 0.1.1 
language-postcss 1.3.1 
language-pug 0.0.21 
linter 2.1.4 
linter-coffeelint 1.2.1 
linter-csslint 1.3.4 
linter-flow 5.6.0 
linter-golint 0.3.0 
linter-pug-lint 0.2.0 
linter-ui-default 1.2.3 
markdown-pdf 1.5.0 
markdown-writer 2.6.4 
merge-conflicts 1.4.4 
minimap 4.28.1 
mocha 0.0.7 
node-debugger 1.10.1 
node-resolver 1.0.1 
php-analyser 0.2.0 
php-cs-fixer 4.1.0 
pigments 0.39.1 
platformio-ide 2.0.0-beta.2 
platformio-ide-terminal 2.5.0 
pretty-json 1.6.3 
run-in-terminal 1.0.0 
switch-header-source 0.22.0 
terminal-panel 1.14.1 
tool-bar 1.1.0 
travis-ci-status 1.3.1 
```

**Comment by DHedgecock** [comment]
Will take a look today.

**Comment by dhedgecock** [comment]
Should be fixed in 2.0.1 👍 , Thanks
